src/Routing/RouteBuilder.php

```diff
@@ -421,7 +426,7 @@ public function resources(string $name, $options = [], $callback = null)
 
         if ($callback !== null) {
             $idName = Inflector::singularize(Inflector::underscore($name)) . '_id';
-            $path = '/' . $options['path'] . '/:' . $idName;
+            $path = '/' . $options['path'] . '/{' . $idName . '}';
```

Comment:
4 concatenations is bit too much, better `sprintf`

Reply:
The total line length is quite reasonable and it's still easy enough to grok what's being done :).